src/webapp/src/client/app/wall/wall.controller.js

```diff
@@ -3,24 +3,38 @@
 
     angular
         .module('simoonaApp.Wall')
+        .constant('minDisplaySizes', {
+            rightSidebar: 992
+        })
         .controller('wallController', wallController);
 
     wallController.$inject = [
         '$rootScope',
+        '$window',
+        '$timeout',
+        'minDisplaySizes',
         'wallRepository'
     ];
 
-    function wallController($rootScope, wallRepository) {
+    function wallController($rootScope, $window, $timeout, minDisplaySizes, wallRepository) {
         /*jshint validthis: true */
         var vm = this;
 
         $rootScope.pageTitle = 'wall.wallTitle';
 
         //init
+        vm.rightBarMinSize = minDisplaySizes.rightSidebar;
+        vm.windowWidth = $window.innerWidth;
         vm.widgetsInfo = {};
         wallRepository.getWidgetsInfo()
-            .then(function(widgetsInfo) { 
+            .then(function (widgetsInfo) {
                 vm.widgetsInfo = widgetsInfo;
             });
+
+        $window.onresize = function () {
+            $timeout(function () {
```

Comment:
I'm not frontend expert, but "responsive" design relying on timeout functions is a bit too hardcore..
If you can try finding some better frontend expert and he could approve this logic - I won't be against it. ;)
@Thomasu , maybe you can take a look?

Reply:
Ok, so it didn't work as expected. This code does now:
```javascript
var resizeTimer;
$(window).on('resize', function () {
	clearTimeout(resizeTimer);
	resizeTimer  =  setTimeout(function () {
		vm.windowWidth  =  $window.innerWidth;
		console.log("new size")
	}, 250);
});
```
It uses the timer before setting window width to the variable, so basically it sets the width when user finishes resizing. 

So to make me clear this is what happens when you resizing without timer: 
![pvz](https://user-images.githubusercontent.com/43700864/65939654-2e20ad00-e42f-11e9-9b7e-9474c5365fee.gif)
Unless it doesn't matter and has no impact on performance. What do you think?